doc/classes/CameraFeed.xml

```diff
@@ -45,10 +45,10 @@
 		<constant name="FEED_RGB" value="1" enum="FeedDataType">
 			Feed supplies RGB images.
 		</constant>
-		<constant name="FEED_YCbCr" value="2" enum="FeedDataType">
+		<constant name="FEED_YCBCR" value="2" enum="FeedDataType">
 			Feed supplies YCbCr images that need to be converted to RGB.
 		</constant>
-		<constant name="FEED_YCbCr_Sep" value="3" enum="FeedDataType">
+		<constant name="FEED_YCBCR_Sep" value="3" enum="FeedDataType">
```

Comment:
"Sep" is not uppercase yet

Reply:
(Note: stopped CI since I know you'll have to amend, don't mind the failed checks.)

Reply:
Already noticed and fixed it :)